replica/tablets.cc

```diff
@@ -104,13 +108,28 @@ tablet_mutation_builder::set_stage(dht::token last_token, locator::tablet_transi
     return *this;
 }
 
+tablet_mutation_builder&
+tablet_mutation_builder::set_session(dht::token last_token, service::session_id session_id) {
+    _m.set_clustered_cell(get_ck(last_token), "session", data_value(session_id.uuid()), _ts);
```

Comment:
We really need DML to work on these tables one day

Reply:
Sometimes it would be an improvement, but other times an update is created dynamically, so having functions that create a mutation for each update and apply them atomically is easier then construct a string for the batch statement that you want to express.